app/views/layouts/_footer.html.erb

```diff
@@ -5,20 +5,61 @@
 
         <div class='row'>
 
-          <div class='col-sm-4'>
-            <p><i class='fa fa-copyright'></i> <%= Date.today.year.to_s %> <%= ENV.fetch('COMPANY_NAME') %>.</p>
-            <ul id='footer-legal' class='list-inline'>
-              <li><%= link_to 'Terms', terms_path %></li>
-              <li><%= link_to 'Privacy', privacy_path %></li>
-              <li><%= link_to "<i class='fa fa-twitter fa-1-25x'></i>".html_safe, ENV.fetch('TWITTER_URL'), target: '_blank' %></li>
-              <li><%= link_to "<i class='fa fa-facebook fa-1-25x'></i>".html_safe, ENV.fetch('FACEBOOK_URL'), target: '_blank' %></li>
-              <li><%= link_to "<i class='fa fa-linkedin fa-1-25x'></i>".html_safe, ENV.fetch('LINKEDIN_URL'), target: '_blank' %></li>
-              <li><%= link_to "<i class='fa fa-medium fa-1-25x'></i>".html_safe, ENV.fetch('MEDIUM_URL'), target: '_blank' %></li>
-            </ul>
+          <div class='col-sm-3'>
+            <div class="container">
```

Comment:
You shouldn't need extra `container` class here – it's generally not a good idea to nest containers in bootstrap. You can nest rows, but containers can cause wackiness with responsive layouts.